src/main/java/io/github/bookcrawler/core/BookStore.java

```diff
@@ -90,6 +94,33 @@ public BooksLinkCrawler crawler() {
         public BookInfoParser parser() {
             return new HelionBookParser();
         }
+    },
+    PUBLIO {
+        @Override
+        public String startUrl() {
+            return "http://www.publio.pl/szukaj,promocja.html?sections=EBOOK";
+        }
+
+        @Override
+        public String domainUrl() {
+            return "http://www.publio.pl";
+        }
+
+        @Override
+        public BookExtractor extractor() {
+            return new PublioBooksExtractor(new JsoupSourceScrapper());
+        }
+
+        @Override
+        public BooksLinkCrawler crawler() {
+
+            return new PublioBookLinksCrawler();
+        }
+
+        @Override
```

Comment:
No, you didn't give points in particular for that. However it made code a little harder to test.